src/components/organisms/CesiumViewer.vue

```diff
@@ -203,6 +203,21 @@ export default defineComponent({
         destination: Cesium.Cartesian3.fromDegrees(-93.849688, 40.690265, 4000000),
       });
       Cesium.Camera.DEFAULT_VIEW_FACTOR = 0;
+
+      // Tooltip handler for showing an entity's properties
+      const handlerToolTips = new Cesium.ScreenSpaceEventHandler(cesiumViewer.value.scene.canvas);
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      handlerToolTips.setInputAction((movement: any) => {
+        const selectedEntity = getSelectedEntityFromPoint(movement.endPosition);
+
+        if (selectedEntity != null) {
+          const properties = selectedEntity.properties.getValue(Cesium.JulianDate.now());
+          console.log(properties);
+          // show pop up tooltip with table of properties
+        } else {
+          // hide popup tooltip
+        }
```

Comment:
@annehaley or @marySalvi,  I need your help having this part show a tooltip above the map with the contents of `properties`